components/Story/AddStory.js

```diff
@@ -0,0 +1,40 @@
+/*
+  Display the new blog creation form
+*/
+import { Button } from 'antd'
+import Link from 'next/link'
+import PropTypes from 'prop-types'
+import React from 'react'
+import { FormattedMessage } from 'react-intl'
+import { connect } from 'react-redux'
+import { Role } from '../../server/services/authorize/role.js'
+// todo if image is not present then use a fallback.
+const AddStory = ({ roles, ...props }) => {
+  if (roles && roles.includes(Role.ADMIN || Role.ORG_ADMIN)) {
```

Comment:
I think the `Role.ADMIN || Role.ORG_ADMIN` operation will always return `Role.ADMIN` as it will return the first truthy value

Reply:
Perhaps
`if (roles && (roles.includes(Role.ADMIN)) || roles.includes(Role.ORG_ADMIN))`

Reply:
Done